include/libopencm3/usb/usbd.h

```diff
@@ -70,24 +70,40 @@ extern const usbd_driver efm32hg_usb_driver;
  *
  * It is required that the 48MHz USB clock is already available.
  *
- * @param driver TODO
+ * @param driver Pointer to an USB driver's information structure.  The
+ *               following drivers are available:
+ *               @li @link st_usbfs_v1_usb_driver @endlink
+ *               @li @link stm32f107_usb_driver @endlink
+ *               @li @link stm32f207_usb_driver @endlink
+ *               @li @link st_usbfs_v2_usb_driver @endlink
+ *               @li @link efm32lg_usb_driver @endlink
+ *               @li @link efm32hg_usb_driver @endlink
```

Comment:
oooh nice doxygen.  is there anyway to j ust link to a type name? we can then make the drivers be members of that instead.  ie, the efm32 driver is already refactored in my local tree.

Reply:
I have no idea, I wrote this with googling and the Doxygen manual open on the side ;)